migrations/env.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import with_statement
+from alembic import context
+from sqlalchemy import engine_from_config, pool
```

Comment:
This migrations file is not used now, when we want to use the database then we can use these files. Should I just delete these too?